tests/integration/knexSnakeCase.js

```diff
@@ -4,7 +4,7 @@ const { Model, knexSnakeCaseMappers } = require('../../');
 const { expect } = require('chai');
 
 module.exports = (session) => {
-  describe('knexSnakeCaseMappers', () => {
+  describe.only('knexSnakeCaseMappers', () => {
```

Comment:
I think you need to remove the only here?

Reply:
oh this embarrassing :smile: I totally forget about that `only`.

Reply:
No worries at all :P